pyecoregen/ecore.py

```diff
@@ -54,6 +55,27 @@ class EcorePackageInitTask(EcoreTask):
     def filename_for_element(package: ecore.EPackage):
         return '__init__.py'
 
+    @staticmethod
+    def imported_classifiers_package(p: ecore.EPackage):
+        """Determines which classifiers have to be imported into given package."""
+        classes = {c for c in p.eClassifiers if isinstance(c, ecore.EClass)}
+
+        references = itertools.chain(*(c.eAllReferences() for c in classes))
+        references_types = {r.eType for r in references}
```

Comment:
Please consider using a generator expression here to conserve memory. In particular since the previous `chain` expression is already a generator and there is no need to instantiate it as real set. The set character of unique entries is still preserved because of the next line, which has to stay a set comprehension.

Reply:
Done! (also a mix between the final code and all the experimentations)

Reply:
Thanks! These changes have not been pushed to the branch yet, did they?

Reply:
I'm pushing them right away, I was waiting in order to package everything in a single commit.